client/src/main/scala/com.gu.contentapi.client/Parameters.scala

```diff
@@ -30,7 +30,7 @@ trait Parameters[Owner <: Parameters[Owner]] { self: Owner =>
   protected def parameterHolder: Map[String, Parameter]
 
   def parameters: Map[String, String] =
-    parameterHolder.mapValues(_.value).collect { case (k, Some(v)) => (k, v.toString) }
+    parameterHolder.mapValues(_.value).toMap.collect { case (k, Some(v)) => (k, v.toString) }
```

Comment:
How come we need a `.toMap` here now?

Reply:
This is because in Scala 2.13 the collections API changed. This is backwards compat, but in the future we will have to change it to `parameterHolder.view.mapValues(_.value)` instead.